device-types/Cisco/C9800-40-K9.yaml

```diff
@@ -14,21 +15,24 @@ interfaces:
   - name: Management Port
     type: 1000base-t
     mgmt_only: true
-  - name: Redundancy Port
+  - name: Redundancy Ethernet Port
     type: 1000base-t
     mgmt_only: true
+  - name: Redundancy SFP Port
```

Comment:
In the configuration, what is the actual ports named? Is it actually called `Redundancy SFP Port`?

Reply:
Sorry, don't have access to a config at the moment, will try and find out the real name of this interface.

Reply:
Please let me know when you have found this information